assets/app/view/game_card.rb

```diff
@@ -42,6 +45,24 @@ def acting?(player)
       acting.include?(player[:id])
     end
 
+    def render_private_join
+      return h(:div, []) unless @is_joining == @gdata['id']
+
+      password_handler = ->(event) { store(:password, Native(event)['target']['value']) }
+      join_handler = lambda do
+        store(:is_joining, nil)
+        join_game(@gdata, @password)
```

Comment:
every time store is called, it rerenders the screen, should we do store(:is_joining, nil, skip: true) here because join_game will call store?